### PR TITLE
minor code cleanup, moved .bat file

### PR DESCRIPTION
**Pull request by tschia**
Made some changes based on some of Wes' code review feedback
Minor changes based on linter

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-.project
 hnclient-v2/.classpath
 hnclient-v2/.settings/org.eclipse.core.resources.prefs
-*.prefs
-mock-point-of-service/.classpath
 hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/tmp/output/hni.txt
+mock-point-of-service/target/
+mock-point-of-service/.idea/*
+mock-point-of-service/.classpath
+moh-hni-bathl7/target/
```
Comment (tschia) [inline .gitignore:16]: cleaned up the gitignore file a bit, added the bathl7 target directory

File: moh-hni-bathl7/bathl7.bat [hunk 2]
```diff
@@ -5,7 +5,7 @@ echo -----------------------------------------
 echo *** BatchHL7 - HL7 Test Facility Starting: ***
 echo -----------------------------------------
 
-java -jar moh-hni-bathl7-1.0.0-SNAPSHOT.jar %*
+java -jar moh-bathl7.jar %*
```
Comment (tschia) [inline moh-hni-bathl7/bathl7.bat:8]: renamed the jar file to be something simple (avoid the risk of the bat file failing if the version changes)

File: moh-hni-bathl7/pom.xml [hunk 3]
```diff
@@ -64,22 +63,21 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<version>2.3.2</version>
+				<version>3.8.1</version>
```
Comment (tschia) [inline moh-hni-bathl7/pom.xml:66]: updated plugin versions to be more recent (and in line with what hns-esb and hns-client use)

File: moh-hni-bathl7/pom.xml [hunk 4]
```diff
@@ -88,6 +86,7 @@
 							<goal>shade</goal>
 						</goals>
 						<configuration>
+							<finalName>moh-bathl7</finalName>
```
Comment (tschia) [inline moh-hni-bathl7/pom.xml:89]: simple jar file name

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java [hunk 5]
```diff
@@ -44,26 +42,29 @@
 	 * @param args arg[0] inputFile args[1] the outputFile args[2] the port
 	 */
 	public static void main(String[] args) {
-		BatHL7Processor batObj = new BatHL7Processor();
+
 		String address = null;
 		String port = null;
-		List<String> hl7TransactionResponse = null;
+		List<String> hl7TransactionResponse;
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java:48]: no need to explicitly assign a value

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java [hunk 6]
```diff
@@ -44,26 +42,29 @@
 	 * @param args arg[0] inputFile args[1] the outputFile args[2] the port
 	 */
 	public static void main(String[] args) {
-		BatHL7Processor batObj = new BatHL7Processor();
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java:47]: moved this to near where it's actually used in the code

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java [hunk 7]
```diff
@@ -74,19 +75,18 @@ public static void main(String[] args) {
 	}
 
 	/**
-	 * @param inputFile
-	 * @param clientAddress
-	 * @param port
-	 * @throws Exception
+	 * @param v2Messages - the v2 messages to send
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java:78]: Javadoc comments should include definitions

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java [hunk 8]
```diff
@@ -74,19 +75,18 @@ public static void main(String[] args) {
...
+	 * @param clientAddress - the HNClient address
+	 * @param port - the HNClient listening port
 	 */
-	private List<String> performTransaction(List<String> lst, String clientAddress, String port) throws Exception {
+	private List<String> performTransaction(List<String> v2Messages, String clientAddress, String port) {
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java:82]: variable name should give an idea of the actual values being stored

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java [hunk 9]
```diff
@@ -159,15 +160,11 @@ private String formatResponse(String request, String response) {
...
+	 * @throws IOException if an I/O error occurs
 	 */
 	public static void deleteFileIfExists(String fileName) throws IOException {
 		Path path = Paths.get(FILE_LOCATION + fileName);
-		try {
-			Files.deleteIfExists(path);
-		} catch (IOException e) {
-			throw e;
-		}
+		Files.deleteIfExists(path);
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java:167]: Don't need to try/catch if we're just immediately throwing the exception

File: moh-hni-bathl7/src/main/java/com/cgi/bathl7/ConnectionHandler.java [hunk 10]
```diff
@@ -95,7 +91,7 @@ protected String socketConnection(String v2Msg, String clientAddress,String port
 				}
 			}
 		}
-		return transactionOut;
+		return null;
```
Comment (tschia) [inline moh-hni-bathl7/src/main/java/com/cgi/bathl7/ConnectionHandler.java:94]: if we reach here this is always null anyways

File: moh-hni-bathl7/src/test/java/com/cgi/bathl7/BatHl7ProcessorTest.java [hunk 11]
```diff
@@ -9,64 +9,49 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import com.cgi.bathl7.BatHL7Processor;
-
 public class BatHl7ProcessorTest {
 
 	@Test
-	public void testReadFile() {
+	public void testReadFile() throws IOException {
```
Comment (tschia) [inline moh-hni-bathl7/src/test/java/com/cgi/bathl7/BatHl7ProcessorTest.java:15]: We can let the junit framework handle the exceptions

File: moh-hni-bathl7/src/test/resources/sample/R031.txt [hunk 12]
```diff
@@ -1,20 +1,20 @@
 MSH|^~\&||BC01000161|RAIGT-PRSN-DMGR|BC00002014|1|xx|||D|2.4
 ZHD|19990605090641|^^00000010|TRAININGAdmin
-PID||9026761259^^^BC^PH^
+PID||900000000^^^BC^PH^
```
Comment (tschia) [inline moh-hni-bathl7/src/test/resources/sample/R031.txt:3]: Better to make it obvious this is fake information

**Review by AnumehaSrivastava05 — APPROVED** [review]

**Pull request merged**